wiki/Tournaments/JHC/JHC_2024/en.md

```diff
@@ -75,6 +335,254 @@ The **JHC 2024** (***Jack House Cup 2024***) is a team–based osu!mania 4K tour
 - Speed Jack
   1. [Hommarju - Crazy Jackpot (\[GB\]V1do) \[Stage 6: JACKpot\]](https://osu.ppy.sh/beatmapsets/2118126#mania/4448636)
 
+## Match results
+
+Sheet-styled match results can be seen through [master sheet](https://docs.google.com/spreadsheets/d/1Qm2BfIc2LFbf59-2RBkqnOde95y2WgoyO_XRNSrvNa4/).
+
+### Grand Finals
+
+Grand Finals:
+
+| \# | Time (UTC) | Team Red |  |  | Team Blue | Match link | VOD link |
+| :-: | :-: | --: | :-: | :-: | :-- | :-: | :-: |
+| 78 | Sunday 10 March 14:00 | wehehe | **7** | 0 | GangWonDuo | [link](https://osu.ppy.sh/community/matches/113019292) | [bilibili](https://www.bilibili.com/video/BV1Nr421p76T/) |
```

Comment:
please follow the [template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md?plain=1#L125-L133)

- the `#` column is ok.
- the line above the table should be the date. remove the `Time (UTC)` column group tables by date, not round
- use `#1` for both link texts